{"uuid": "82ad0bd1-5c3b-4c58-a3ac-9c993e89d662", "children": ["b9c6549a-7fba-419b-bcf5-bdc3005af085"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "AttributeError: 'function' object has no attribute 'get'\n", "trace": "  File \"/Users/chethangopal/Desktop/Learning/.venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Users/chethangopal/Desktop/Learning/.venv/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/Users/chethangopal/Desktop/Learning/.venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/chethangopal/Desktop/Learning/.venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1122, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/chethangopal/Desktop/Learning/.venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 877, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/Users/chethangopal/Desktop/Learning/tests/conftest.py\", line 29, in driver\n    driver.get(base_url)\n    ^^^^^^^^^^\n"}, "start": 1716712727482, "stop": 1716712727483}], "start": 1716712727482, "stop": 1716712727496}